astra_ros/ImageStreamMode image_stream_mode
---